src/app/index.tsx

```diff
@@ -31,6 +32,7 @@ async function checkForInMemoryKeys() {
 }
 
 async function renderApp() {
+  logger.info('Interested in joining our team? Email us: wallet@hiro.so');
```

Comment:
```suggestion
  logger.info('Interested in joining our team? Email us: wallet@hiro.so');
```

Only in production please. I do not want to read this every time I reload the app.

Reply:
I can follow up on this as I was going to add the security warning (can be one msg)

Reply:
Can we remove this section actually. Let's keep the dev tools warning for security and keep the job push to html file.